Header header
float64 r
float64 lx
float64 ly
float64 T
float64 N
